Find a functionally equivalent code:android.content.RestrictionEntry.getChoiceValues:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the list of possible string values set earlier.
 * @return the list of possible values.
 */

@@ -1,3 +1,3 @@
 {
-    return values;
+    return mChoiceValues;
 }

